src/range_proof/mod.rs

```diff
@@ -613,7 +714,9 @@ mod tests {
 
     #[test]
     fn create_and_verify_n_64_m_8() {
-        singleparty_create_and_verify_helper(64, 8);
+        // XXX disable this test since it's failing with
+        //     'attempt to negate with overflow'
+        //singleparty_create_and_verify_helper(64, 8);
```

Comment:
This seems kind of worrying (in the sense that it may point at a problem in the rest of the source code) -- is it reproducible? I tried to reproduce on my machine but I wasn't able to.

Reply:
Weird, I thought I reverted this.  It was failing locally, but not remotely in CI.  

Reverted now.